### DISCUSSION TITLE
Is it Laravel Octane compatible?

### DISCUSSION THREAD

**Discussion by TC74** [lunarphp/lunar#1555] [Q&A]
It is Laravel Octane FrankenPhp compatible? It would be a waste not to optimise its performance.

https://laravel.com/docs/10.x/octane
https://frankenphp.dev/

**Comment by eugenefvdm** [comment]
Hi there @TC74 ,

Your comment seems misdirected? This is the Lunar PHP e-commerce platform?

**Reply by glennjacobs** [reply]
I don't think there's any mistake. 

We haven't performed any tests to date, but would welcome contributions.

**Reply by eugenefvdm** [reply]
Ah apologies! I misread that sentence.

**Comment by lguichard** [comment]
Lunar is compatible with Octane. I used Octane with FrankenPHP on my project that's not problem. But Lunar don't have official support.

**Reply by glennjacobs** [reply]
We could do with some tests for this somehow. I've not looked into it.

**Comment by Alph44rchitect** [comment]
I managed to get Lunar 1.x and Octane 2.4 running together in Laravel 11 with Sail (Docker), but I noticed that the performance has unfortunately deteriorated significantly. I compared `sail artisan serve` and `sail artisan octane:start`.

I found that the **boot time** in my case increased from about **8ms** (without Lunar) to about **160ms** (with Lunar) when using `octane`, while it increased from about **10ms** (without Lunar) to **40ms** (with Lunar) when using `serve`.
